build-tools/Xamarin.Android.Tools.BootstrapTasks/result-packaging.targets

```diff
@@ -25,9 +25,7 @@
     <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\TestResult-*.xml" />
     <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\compatibility\*" />
     <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\logcat*" />
-    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\**\*.binlog" />
-    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\**\*.log" />
-    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\**\screenshot.png" />
+    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\temp\**\*" />
```

Comment:
This change seems to have *removed* the `temp` directory within the output files.

Consider https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3087455&view=artifacts > **mac-msbuild-test-results** > [**xa-test-results-10.0.99.150_7ad0256-Darwin-Release.zip**](https://artprodscussu1.artifacts.visualstudio.com/A011b8bdf-6d56-4f87-be0d-0092136884d9/0bdbc590-a062-4c3f-b0f6-9383f67865ee/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2RldmRpdi9wcm9qZWN0SWQvMGJkYmM1OTAtYTA2Mi00YzNmLWIwZjYtOTM4M2Y2Nzg2NWVlL2J1aWxkSWQvMzA4NzQ1NS9hcnRpZmFjdE5hbWUvbWFjLW1zYnVpbGQtdGVzdC1yZXN1bHRz0/content?format=zip&subPath=%2Fxa-test-results-10.0.99.150_7ad0256-Darwin-Release.zip):

```bash
$ ls -l xa-test-results-10.0.99.150_7ad0256-Darwin-Release
total 25264
drwxr-xr-x   6 jon  staff       192 Sep 26 20:26 CheckPerformanceOfManagedParser
-rw-r--r--@  1 jon  staff  12894710 Sep 27  2019 TestResult-MSBuildTests-Release.xml
-rw-r--r--@  1 jon  staff     36643 Sep 27  2019 TestResult-MSBuildTestsCommercial-Release.xml
drwxr-xr-x  20 jon  staff       640 Sep 26 20:26 packages
```

I believe that `CheckPerformanceOfManagedParser` should be within `temp`, but isn't.  I don't know why `packages` is in there at all.  (It might be *helpful* for it to be there, but I don't currently understand *why* it is in there.)

Reply:
Whatever it's doing here, seems completely wrong. Looking into it.